llvm/lib/Analysis/LoopInfo.cpp

```diff
@@ -999,6 +999,18 @@ void llvm::printLoop(Loop &L, raw_ostream &OS, const std::string &Banner) {
     return;
   }
 
+  if (forcePrintFuncIR()) {
```

Comment:
```suggestion
  if (forcePrintModuleIR() || forcePrintFuncIR()) {
```

I'm unsure as to what semantics we would want here. In your experience, is it helpful to have the module printed out followed by the function?

Reply:
I usually want to run -print-after-all, split the output into files, and diff files corresponding to consecutive passes. If the entire module _is_ dumped, the diff is less clean. Hence just the func IR.

With the suggested change, the function will get dumped in _addition_ to the module when -print-module-scope is specified, I think. Since the function is a part of the module, this seems redundant.

Reply:
Discussed offline. 👍